extensions/arbext.php

```diff
@@ -1,422 +0,0 @@
-<ol>
```

Comment:
Did you do a 'make' in extensions/ to regenerate these files? Something appears to have messed up as AFAICT from the diff viewer, the .php files are empty.